### ISSUE TITLE
Migrate to compose multiplatform

### ISSUE THREAD

**Issue by Deorigami** [touchlane/gridpad-android#25]
is there a plan to migrate this to compose multiplatform ?

**Comment by landarskiy** [comment]
Hi @Deorigami , and thanks for your interest in this library.

We've never really thought about that, but it's a good idea. I'll add it to my roadmap. I can't guarantee a specific date, but I think we can expect the library to be available in the Compose Multiplatform world as well.

**Comment by Deorigami** [comment]
currently i just download the files in the module and then put it on common module on my project and its work

**Comment by landarskiy** [comment]
Yes, the library doesn't use anything Compose Multiplatform doesn't support. It's more of an infrastructure kind of work.

**Comment by Geka000** [comment]
Hello @landarskiy. 
I'm using your library in my Android project and trying to port it to compose multiplatform. 
Are there any updates on migration to multiplatform? 
I've just tried to fork and make migration by myself. The code migration is easy, but publication is where things get complicated.

**Comment by prsltdpersonal** [comment]
Hi.

I'd very much like to try this on compose multiplatform. I'm a newbie to compose/multiplatform. When I add the dependencies, I get an error that it couldn't resolve com.touchlane:gridpad:1.1.2 (I also tried 1.1.0) @Deorigami  mentions downloading the files etc, please could you give details how to do this ?
